utils/__init__.py

```diff
@@ -397,20 +398,17 @@ def ansi_to_html(text):
         if len(text_message) > 0:
             text = text_message
             text = bytes(text[0], "utf-8").decode("unicode_escape")
-        elif 'The COMBINE/OMEX did not execute successfully:' in text:
-            text = text # to deal with remote error message
-        else:
-            text = text.replace('|', '')
-            return text
+        # elif 'The COMBINE/OMEX did not execute successfully:' in text:
+        #     text = text # to deal with remote error message
 
         text = text.replace('|', '')
 
         # # for any text with "<*>" remove "<" as well as ">" but leave wildcard text *
         text = re.sub(r'<([^>]*)>', r'\1', text)
 
         # replace color codes with html color codes
```

Comment:
This comment seems out of date now, as the escape codes are being removed rather than replaced with HTML now?